DependencyInjection/AbstractSonataAdminExtension.php

```diff
@@ -43,7 +43,7 @@ protected function fixTemplatesConfiguration(array $configs, ContainerBuilder $c
                         'datetime' => 'SonataAdminBundle:CRUD:list_datetime.html.twig',
                         'text' => 'SonataAdminBundle:CRUD:list_string.html.twig',
                         'textarea' => 'SonataAdminBundle:CRUD:list_string.html.twig',
-                        'email' => 'SonataAdminBundle:CRUD:list_string.html.twig',
+                        'email' => 'SonataAdminBundle:CRUD:list_email.html.twig',
```

Comment:
> does this keep the same behaviour or is this a BC break?

@OskarStark Instead of `admin@example.com` will be `<a href="mailto:admin@example.com">admin@example.com</a>`